### PR TITLE
DSRS v0.2.0

### PR DESCRIPTION
**Pull request by mdreyesm**
# DSRS v0.2.0
- Updated package versions on all images.
- Barlow Pass support for PMEM enabled images.
- Openshift support for Spark images.
- Helm chart for deploying Spark clusters.

### CODE REVIEW

File: dsrs/redis/README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
-# Data Services Reference Stack(DSRS) - Redis* 
+# Data Services Reference Stack(DSRS) - Redis
```
Comment (luisfponce) [inline dsrs/redis/README.md]: @mdreyesm Hi Miguel, probably you might want to put back `*` to reference to `
*\*Other names and brands may be claimed as the property of others*` legend for Redis.

This  of course depends on Legal feedback, ryt?

File: dsrs/memcached/README.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
-# Data Services Reference Stack(DSRS) - Memcached* 
+# Data Services Reference Stack(DSRS) - Memcached
```
Comment (luisfponce) [inline dsrs/memcached/README.md]: Same `*` here

File: dsrs/spark/README.md [hunk 3]
```diff
@@ -1,8 +1,8 @@
-# Data Services Reference Stack(DSRS) – Apache Spark*  
+# Data Services Reference Stack(DSRS) – Spark
```
Comment (luisfponce) [inline dsrs/spark/README.md]: Same `*` here

**Review by luisfponce — COMMENTED** [review]
lgtm, just couple of "Documents" feedback

File: dsrs/redis/README.md (hunk 1 above)
Comment (mdreyesm) [reply dsrs/redis/README.md]: done

File: dsrs/memcached/README.md (hunk 2 above)
Comment (mdreyesm) [reply dsrs/memcached/README.md]: done

File: dsrs/spark/README.md (hunk 3 above)
Comment (mdreyesm) [reply dsrs/spark/README.md]: done

**Comment by luisfponce** [comment]
lgtm

**Pull request merged**